js/packages/seed-vault/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "../../tsconfig.json",
+    "include": ["src"],
+    "compilerOptions": {
+        "declarationDir": "./lib/types",
+        "outDir": "lib/esm"
```

Comment:
I don't think you need CommonJS and ESM builds since this is exclusively for React Native. Just use ESM.